{{cookiecutter.project_shortname}}/invenio.cfg

```diff
@@ -47,11 +47,12 @@ APP_DEFAULT_SECURE_HEADERS = {
             "'self'",
             'fonts.googleapis.com',  # for fonts
             '*.gstatic.com',    # for fonts
+            'data:' # for fonts
             "'unsafe-inline'",  # for inline scripts and styles
+            "'unsafe-eval'", # for webpack build
```

Comment:
I noticed the CSP warning about this one, but I didn't see anything break from it. What was affected? (didn't see the `data` one though)

Reply:
this is webpack when it injects react in the dom and the data is when loading some fonts through the `data` scheme. Still the `unsafe-eval` is not a best practice and I saw people having issues due to webpack as it is using the method of string evaluation as javascript. We need to dig more into this. I will create an issue for that!

Reply:
Yeah, I've been reluctant to allow the unsafe-eval. But webpack should be at build time. It shouldn't be on a served page right?